data/blog/webview2.md

```diff
@@ -0,0 +1,108 @@
+| Source Available on GitHub                | Yes             | No                           |
+| Shares Edge/Chrome DLLs                   | No              | No                           |
+| Shared Runtime Between Applications       | No              | Optional                     |
+| Application APIs                          | Yes             | No                           |
+| Node.js                                   | Yes             | No                           |
+| Sandbox                                   | Optional        | Always                       |
+| Requires a Separate Application Framework | No              | Yes                          |
+| Supported Platforms                       | Mac, Win, Linux | Win (Mac on roadmap)         |
+
+## Performance Discussion
+
+Now the big question:
+_Will your app use 50% less memory by switching between one embedding of Chromium and another?_
+Unfortunately not, but that would have been rad.
+
+We created the [scaffolding to compare some simple apps built using Electron, WPF+WebView2, and .NET+WebView2](https://github.com/crossplatform-dev/xplat-challenges/blob/main/results.md).
+Our goal was to find any egregious differences, rather than deep-dive on micro-benchmarks.
+The source code and detailed results are available in the [xplat-challenges](https://github.com/crossplatform-dev/xplat-challenges) repository,
+so you’re free to play around with the frameworks yourself.
+
+### A Note About Inter-Process Communication (IPC)
+
+In Chromium, the browser process acts as an IPC broker between sandboxed renderers and the rest of the system.
+While Electron allows unsandboxed render processes, many apps choose to enable the sandbox for added security.
+WebView2 always has the sandbox enabled, so for most Electron and WebView2 apps IPC can heavily impact overall performance.
+
+Even though Electron and WebView2 have a similar process models, their underlying IPC implementation is different.
+Communicating between JavaScript and C++ or C# requires [marshalling](https://en.wikipedia.org/wiki/Marshalling_(computer_science)),
+most commonly into a JSON string. This is an expensive operation and IPC bottlenecks can negatively impact performance.
+
+When sending JavaScript objects between Electron's main and renderer process,
+Electron will use [the structured clone algorithm](https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API/Structured_clone_algorithm) which is [significantly faster](https://github.com/crossplatform-dev/xplat-challenges/blob/main/results.md#ipc) even when using context isolation for increased security.
+Electron also supports direct IPC between any two processes via the [MessagePorts](https://www.electronjs.org/docs/latest/performance/message-ports/) API,
+which also utilizes structured clone.
+Applications which leverage this can avoid paying the JSON-serialization tax when sending objects between processes.
```

Comment:
Starting from edge 93, WV2 will use CBOR protocol for network events

Reply:
Do you have any links I can point folks to?

Reply:
I added a note. It currently links to Wikipedia, but I'm happy to point it elsewhere if you have more info.